client/src/pages/DocumentPage.jsx

```diff
@@ -131,9 +140,21 @@ export default function DocumentPage() {
           >
             Back To Homepage
           </button>
-          <button className="document-button share-button">Share</button>
-          <div className="user-avatar">DM</div>{" "}
-          {/* PLACEHOLER will change when userAUTH is done */}
+          <button
+            className="document-button share-button"
+            onClick={() => setShowShareModal(true)}
+          >
+            Share
+          </button>
+          <div className="user-avatar">
+            {loading ? (
+              "loading"
+            ) : user?.image ? (
+              <img src={user.image} alt="User Avatar" />
+            ) : (
+              "No Avatar"
+            )}
```

Comment:
Once you have more than 2 choices, it becomes a bit hard to read, so I suggest pulling it out into its own method with explicit `if/else` but thats just a nit going forward.